h/static/scripts/group-forms/components/EditGroupMembersForm.tsx

```diff
@@ -25,10 +35,22 @@ async function fetchMembers(
     signal,
   });
   return members.map(member => ({
+    userid: member.userid,
     username: member.username,
+    canDelete:
+      member.actions.includes('delete') && member.userid !== currentUserid,
   }));
 }
 
+async function removeMember(api: APIConfig, userid: string) {
+  const { url: urlTemplate, method, headers } = api;
+  const url = urlTemplate.replace(':userid', encodeURIComponent(userid));
```

Comment:
I plan to add a utility to handle replacing path params in a separate PR. We have suitable code in the lms app for this.